assets/src/components/ladder.tsx

```diff
@@ -220,23 +218,25 @@ const LadderTimepoint = ({
       y={y}
       textAnchor="middle"
       dominantBaseline="middle"
+      data-tip={timepoint.name}
```

Comment:
If the name is missing, this would be `null`. Does react tooltip handle that well?

Reply:
Yes, you can see this for "CENJF" on route 502: there's no tooltip if there's no content.